bundles/org.openhab.binding.doorbird/README.md

```diff
@@ -0,0 +1,239 @@
+# Doorbird Binding
+
+Binding for Doorbird D101 and D210x video doorbells.
+
+## Supported Things
+
+Two thing types are supported:
+
+- Doorbird D101
+- Doorbird D210x
+
+## Discovery
+
+Auto-discovery is not supported at this time.
+
+## Thing Configuration
+
+The following configuration parameters are available on the Doorbird thing:
+
+| Parameter | Required/Optional | Description |
+|-----------|-------------------|-------------|
+| ID | Required | The ID number of the Doorbird device. This is usually a single digit (e.g. 1). |
```

Comment:
Please check formatting

Reply:
This does format correctly (see here https://github.com/mhilbush/openhab2-addons/tree/doorbird-new/bundles/org.openhab.binding.doorbird#thing-configuration).

And it seems to have the benefit of not being tedious to maintain with all the extra white space.

Reply:
I think there are different opinions :-). I prefer it aligned because it's easier to see in which columns the text is. AFAIK there is no strict guildeline.